supervisor/resolution/fixups/create_full_snapshot.py

```diff
@@ -0,0 +1,26 @@
+"""Helpers to check and fix issues with free space."""
+import logging
+
+from ..const import ContextType, SuggestionType
+from ..data import Suggestion
+from .base import FixupBase
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class FixupCreateFullSnapshot(FixupBase):
+    """Storage class for fixup."""
+
+    async def process_fixup(self, suggestion: Suggestion) -> None:
```

Comment:
This should have a FREE_SPACE job condition in case something changed between when the suggestion was made and the action was executed.

Reply:
The snapshot job have it